ERROR - 2022-11-08 17:16:21 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'pw_example' in /var/www/html/codeigniter4/system/Database/MySQLi/Connection.php:171
Stack trace:
#0 /var/www/html/codeigniter4/system/Database/MySQLi/Connection.php(171): mysqli->real_connect('db', 'root', '1q2w3e4r5t', 'pw_example', 3306, '', 0)
#1 /var/www/html/codeigniter4/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /var/www/html/codeigniter4/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 /var/www/html/codeigniter4/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 /var/www/html/codeigniter4/system/Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 /var/www/html/codeigniter4/system/BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#6 /var/www/html/codeigniter4/app/Controllers/Client.php(13): CodeIgniter\BaseModel->findAll()
#7 /var/www/html/codeigniter4/system/CodeIgniter.php(927): App\Controllers\Client->listClients()
#8 /var/www/html/codeigniter4/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Client))
#9 /var/www/html/codeigniter4/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /var/www/html/codeigniter4/public/index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'pw_example' in /var/www/html/codeigniter4/system/Database/MySQLi/Connection.php:215
Stack trace:
#0 /var/www/html/codeigniter4/system/Database/BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /var/www/html/codeigniter4/system/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 /var/www/html/codeigniter4/system/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 /var/www/html/codeigniter4/system/Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 /var/www/html/codeigniter4/system/BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
#5 /var/www/html/codeigniter4/app/Controllers/Client.php(13): CodeIgniter\BaseModel->findAll()
#6 /var/www/html/codeigniter4/system/CodeIgniter.php(927): App\Controllers\Client->listClients()
#7 /var/www/html/codeigniter4/system/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Client))
#8 /var/www/html/codeigniter4/system/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /var/www/html/codeigniter4/public/index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-08 17:16:21 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'pw_example'
in SYSTEMPATH/Database/BaseConnection.php on line 418.
 1 SYSTEMPATH/Database/BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `clients`', [], false)
 3 SYSTEMPATH/Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH/BaseModel.php(585): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH/Controllers/Client.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH/CodeIgniter.php(927): App\Controllers\Client->listClients()
 7 SYSTEMPATH/CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Client))
 8 SYSTEMPATH/CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
